.gitignore

```diff
@@ -59,3 +59,30 @@ typings/
 
 # next.js build output
 .next
+
+# General
+.DS_Store
+.AppleDouble
+.LSOverride
+
+# Icon must end with two \r
+Icon

+
+# Thumbnails
+._*
+
+# Files that might appear in the root of a volume
```

Comment:
Why would all these excludes for root/AFP be necessary? Just curious.

Reply:
Iuse the template from https://marketplace.visualstudio.com/items?itemName=codezombiech.gitignore and they seem to work for me just fine - of course its 90% not relevant for the project but they are good defaults